### PR TITLE
Add How-To Integrate `k8s-operator` with `etcd`

### PR DESCRIPTION
**Pull request by mateoflorido**
## Overview
This pull request introduces a How-To guide on integrating the Canonical Kubernetes operator with an external etcd datastore.

### CODE REVIEW

File: docs/src/charm/howto/etcd.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+```bash
+juju integrate etcd easyrsa
+```
+
+## Deploying Canonical Kubernetes
+Deploy the control plane units of Canonical Kubernetes with the command:
+
+```bash
+juju deploy k8s --config datastore=etcd -n 3
+```
```
Comment (addyess) [inline docs/src/charm/howto/etcd.md:77]: i don't know if it's possible, but we should really encourage `juju expose k8s` in our How-Tos.  Without it people will be frustrated with the experience on public clouds

File: docs/src/charm/howto/etcd.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+```bash
+➜  ~ juju status
+Model       Controller  Cloud/Region    Version  SLA          Timestamp
+my-cluster  canosphere  vsphere/Boston  3.4.0    unsupported  16:15:19-05:00
+
+App      Version  Status  Scale  Charm    Channel      Rev  Exposed  Message
+easyrsa  3.0.1    active      1  easyrsa  stable        55  no       Certificate Authority connected.
+etcd     3.4.22   active      3  etcd     stable       760  no       Healthy with 3 known peers
+k8s      1.29.3   active      3  k8s      latest/edge   31  no       Ready
```
Comment (addyess) [inline docs/src/charm/howto/etcd.md:107]: If exposed, status would look like this

```suggestion
k8s      1.29.3   active      3  k8s      latest/edge   31  yes      Ready
```

**Review by addyess — APPROVED** [review]
This is really nice

**Comment by eaudetcobello** [comment]
Good stuff!

**Review by evilnick — APPROVED** [review]
:cake: :tada:

**Pull request merged**